samples/kubernetes/end2end/docs/05-prometheus.md

```diff
@@ -10,9 +10,10 @@ Deploy the PV and PVC YAML files.
 kubectl apply -f prometheus/persistence.yaml
 kubectl apply -f prometheus/alert-persistence.yaml
 ```
-Install the Prometheus chart.
+Install the Prometheus chart, in this sample we will use version 17.0.0.
```

Comment:
Shouldn't the comma here be a semicolon or period, @rosemarymarano ?

Reply:
@russgold Yes, you are correct. Either will do, a semi-colon and keep the rest of the sentence the same, or a period and make it into two sentences.

Reply:
fixed